### PR TITLE
Enable caching in stage command.

### PR DESCRIPTION
**Pull request by danxmoran**
The latest combination of SBT and the native-packager plugin seem to handle this intelligently.

### CODE REVIEW

File: src/sbt-test/sbt-plugins/cache-key/test [hunk 1]
```diff
@@ -16,10 +16,17 @@ $ copy-file webservice/target/universal/stage/conf/cacheKey.Sha1 cacheKey2.Sha1
 > assertCacheKeysEqual
 
 # cache key should change when library dependencies change
+-$ exists webservice/target/universal/stage/lib/org.apache.derby.derby-10.4.1.3.jar
```
Comment (jkinkead) [inline src/sbt-test/sbt-plugins/cache-key/test]: Can you also add a test that verifies that removing files from [one of the mapped directories](https://github.com/danxmoran/sbt-plugins/blob/2d20916084f23ba85b2e65082f7b7110dbc10496/src/main/scala/org/allenai/plugins/DeployPlugin.scala#L147-L152) is also handled?

**Comment by jkinkead** [comment]
One request-for-test. :)

**Comment by danxmoran** [comment]
Test added!

**Comment by jkinkead** [comment]
Awesome, LGTM!

**Pull request merged**